src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/Tools/Move-AzureRmVhdVM.ps1

```diff
@@ -0,0 +1,237 @@
+<#
+.ExternalHelp AzureRM.Compute.ManagedService-help.xml
+#>
+
```

Comment:
If there is a dependency on the local Hyper-V PowerShell module, it should go here in #Requires

Reply:
Also, if this is going to depend on the Compute cmdlets and/or the Profile cmdlets, those dependencies should go here

Reply:
CmdletBinding?

Reply:
OutputType?

Reply:
ok